plugins/woocommerce-admin/client/marketing/data-multichannel/actions.ts

```diff
@@ -38,9 +43,41 @@ export const receiveRecommendedChannelsError = ( error: ApiFetchError ) => {
 	};
 };
 
+export const receiveCampaignsSuccess = ( success: {
+	payload: Array< Campaign >;
+	error: boolean;
+	meta: {
+		page: number;
+		perPage: number;
+		total?: number;
+	};
+} ) => {
+	return {
+		type: TYPES.RECEIVE_CAMPAIGNS_SUCCESS,
+		...success,
+	};
+};
+
+export const receiveCampaignsError = ( error: {
+	payload: ApiFetchError;
+	error: boolean;
+	meta: {
+		page: number;
+		perPage: number;
+		total?: number;
+	};
```

Comment:
💅  In the line of my previous comment. It has some success fields. Even is an error